homeassistant/components/eheimdigital/config_flow.py

```diff
@@ -0,0 +1,93 @@
+"""Config flow for EHEIM Digital."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from aiohttp import ClientError
+from eheimdigital.hub import EheimDigitalHub
+import voluptuous as vol
+
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+from homeassistant.config_entries import SOURCE_USER, ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import selector
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema(
+    {vol.Required(CONF_HOST, default="eheimdigital"): selector.TextSelector()}
+)
+
+
+class EheimDigitalConfigFlowHandler(ConfigFlow, domain=DOMAIN):
+    """The EHEIM Digital config flow."""
+
+    data: dict[str, Any] = {}
```

Comment:
Don't create mutable objects like dicts and lists on class level as they are shared between every instance of this object and that can lead to side effects and hard to track down bugs

Reply:
Yeah of course, I overlooked that.